src/passes/DeadArgumentElimination.cpp

```diff
@@ -237,11 +284,25 @@ struct DAE : public Pass {
     // This set tracks the functions for whom calls to it should be modified.
     std::unordered_set<Name> callTargetsToLocalize;
 
+    // As we optimize, we mark things as stale.
```

Comment:
Can you say more about when we mark things stale? What is the effect of marking things stale?

Reply:
See the comment on line 59, above `stale = true`:

> Whether this needs to be recomputed. This begins as true for the first computation, and we reset it every time we touch the function.

Stale info needs to be recomputed. Info becomes stale when changes happen. Is there a better term perhaps?

Reply:
I guess I'm just missing a lot of context for what this code is doing to begin with, so I'm having trouble understanding the changes. I'll go read the existing code more to orient myself.